ecom-mockman/src/Components/Navbar.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import "./Navbar.css"
+import ProfileImg from "../images/Profile-navbar.png"
+import CartIcon from "../images/cart-icon.png"
+import {NavLink} from "react-router-dom"
+import StormIcon from '@mui/icons-material/Storm';
+import MenuIcon from '@mui/icons-material/Menu';
+
+function Navbar() {
+  return (
+    <div>
+             
+        {/* <!-- Nav --> */}
+    <div class="header-wrapper">
```

Comment:
in react  className is used  instead of class

Reply:
my code had some problems, so i had to directly import from my html project, it needs a refactor for React for sure. thanks for pointing it out.